src/utils/validation.js

```diff
@@ -0,0 +1,13 @@
+export const validateEmail = (email) => {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    return emailRegex.test(email);
+  };
+  
+  export const validatePassword = (password) => {
+    return password.length >= 6;
+  };
+  
+  export const validateConfirmPassword = (password, confirmPassword) => {
+    return password === confirmPassword;
+  };
```

Comment:
We don't need to have this here, there's only one place it's been used. Just do the check in the registration component